### PR TITLE
Update type and comment in Timeoutmarshaller

### PR DESCRIPTION
**Pull request by carl-mastrangelo**

### CODE REVIEW

File: core/src/main/java/io/grpc/internal/GrpcUtil.java [hunk 1]
```diff
@@ -436,7 +436,8 @@ public void close(ScheduledExecutorService instance) {
    */
   @VisibleForTesting
   static class TimeoutMarshaller implements Metadata.AsciiMarshaller<Long> {
-    private static final BiMap<TimeUnit, Character> UNITS =
+    // Bimaps have consistent iteration order.
```
Comment (ejona86) [inline core/src/main/java/io/grpc/internal/GrpcUtil.java]: s/Bimaps/ImmutableBiMaps/

The Bimap iterface does not define a consistent iteration order.

**Comment by ejona86** [comment]
@carl-mastrangelo, one comment, but otherwise LGTM

**Comment by ejona86** [comment]
(And checkstyle is failing due to unused import)

**Comment by carl-mastrangelo** [comment]
PTAL, I changed this to use Map in case we end up having to give up the Immutable classes for android.

**Comment by ejona86** [comment]
@carl-mastrangelo, LGTM

**Pull request merged**